Files/Filesystem/FilesystemOperations/Helpers/FilesystemHelpers.cs

```diff
@@ -602,13 +602,12 @@ public async Task<ReturnResult> CopyItemsFromClipboard(DataPackageView packageVi
             {
                 ReturnResult returnStatus = ReturnResult.InProgress;
 
-                List<string> destinations = new List<string>();
+                var destinations = new List<string>();
+                List<ShellFileItem> binItems = await recycleBinHelpers.EnumerateRecycleBin();
```

Comment:
Nope xD Now it queries the recycle bin even if no items are under the recyclebin

Reply:
😅 I reverted to the original keeping the fix